x-pack/legacy/plugins/siem/public/containers/network_top_n_flow/index.tsx

```diff
@@ -132,7 +132,7 @@ class NetworkTopNFlowComponentQuery extends QueryTemplatePaginated<
   }
 }
 
-const mapStateToProps = (state: State, { flowTarget, id = ID }: OwnProps) => {
+const mapStateToProps = (state: State, { flowTarget, id = ID + flowTarget }: OwnProps) => {
```

Comment:
nit: I would move this to just an interpolation value `${ID}-${flowTarget}` with a dash separator. Then you get the id's back as dash separated like so:

```
networkTopNFlowQuery-source
networkTopNFlowQuery-destination
```

Which will be a bit more debugger friendly and follow the other conventions of id's we have.